src/panels/config/integrations/integration-panels/bluetooth/bluetooth-advertisement-monitor.ts

```diff
@@ -140,6 +144,16 @@ export class BluetoothAdvertisementMonitorPanel extends LitElement {
           sortable: true,
           defaultHidden: true,
         },
+        time: {
+          title: localize("ui.panel.config.bluetooth.last_seen"),
+          filterable: false,
+          sortable: true,
+          defaultHidden: false,
+          template: (ad) => {
+            const date = new Date(ad.time * 1000);
+            return html`${relativeTime(date, locale)}`;
```

Comment:
Should we use `<ha-relative-time>` component for live update?

Reply:
I can certainly change it to that if you prefer, but I guess most of the other config panels just use `relativeTime()`

Reply:
Let's indeed use `<ha-relative-time>` that makes sure it keeps up to date, when we use `relativeTime()` it will be dependant on when the data is updated.

Reply:
Done.